build/teamcity/internal/release/process/make-and-publish-build-tagging.sh

```diff
@@ -28,51 +32,58 @@ fi
 
-  build_name:      $build_name
-  release_branch:  $release_branch
-  is_custom_build: $is_custom_build
-  gcr_repository:  $gcr_repository
+  build_name:          $build_name
+  release_branch:      $release_branch
+  is_customized_build: $is_customized_build
 
 EOF
 tc_end_block "Variable Setup"
 
-
-tc_start_block "Tag the release"
-git tag "${build_name}"
-tc_end_block "Tag the release"
-
-tc_start_block "Push release tag to github.com/cockroachlabs/release-staging"
-github_ssh_key="${GITHUB_COCKROACH_TEAMCITY_PRIVATE_SSH_KEY}"
 configure_git_ssh_key
-git_wrapped push ssh://git@github.com/cockroachlabs/release-staging.git "${build_name}"
-tc_end_block "Push release tag to github.com/cockroachlabs/release-staging"
-
 
-tc_start_block "Tag docker image as latest-build"
-# Only tag the image as "latest-vX.Y-build" if the tag is on a release branch
-# (or master for the alphas for the next major release).
-if [[ -n "${release_branch}" ]] ; then
-  log_into_gcloud
-  gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-${release_branch}-build"
-fi
-if [[ "$TC_BUILD_BRANCH" == "master" ]]; then
-  log_into_gcloud
-  gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-master-build"
-fi
+if [[ -z "${is_customized_build}" ]] ; then
+  tc_start_block "Tag the release"
+  git tag "${build_name}"
+  tc_end_block "Tag the release"
+
+  tc_start_block "Push release tag to github.com/cockroachlabs/release-staging"
+  git_wrapped push ssh://git@github.com/cockroachlabs/release-staging.git "${build_name}"
+  tc_end_block "Push release tag to github.com/cockroachlabs/release-staging"
+
+
+  tc_start_block "Tag docker image as latest-build"
+  # Only tag the image as "latest-vX.Y-build" if the tag is on a release branch
+  # (or master for the alphas for the next major release).
+  if [[ -n "${is_release_build}" ]] ; then
+    log_into_gcloud
+    gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-${release_branch}-build"
+  fi
+  if [[ "$TC_BUILD_BRANCH" == "master" ]]; then
+    log_into_gcloud
+    gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-master-build"
+  fi
 tc_end_block "Tag docker image as latest-build"
-
+fi
 
 # Make finding the tag name easy.
 cat << EOF
 
 
 Build ID: ${build_name}
 
+The binaries are available at:
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-arm64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.darwin-10.9-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.windows-6.2-amd64.zip
+
+Pull the docker image by:
+  docker pull us-docker.pkg.dev/cockroach-cloud-images/cockroachdb-customized/cockroach-customized:$build_name
```

Comment:
How about making this point at the right place for customized and dry run builds?